measurements/database.py

```diff
@@ -16,6 +17,14 @@
 
 from measurements.config import request_id
 
+from flask import Flask
+from prometheus_flask_exporter import PrometheusMetrics
+from prometheus_client import Gauge
+
+app = Flask(__name__)
+registry = PrometheusMetrics(app).registry
+query_time = Gauge("query", "query", ["hash",], registry=registry)
```

Comment:
I think this code should be moved into the app initialisation. I don't think you should be duplicating the Flask app instance, as the initialisation is meant to be done only once.